### DISCUSSION TITLE
Swipe on the open article to go to the next or previous one.

### DISCUSSION THREAD

**Discussion by wyntrson** [spacecowboy/Feeder#188] [Ideas]
The back and forth is such a painful thing to do when reading a lot of articles.
It would be great to include a feature to swipe left or right for previous or next articles.

**Comment by spacecowboy** [comment]
Often requested.

Personally I hate this feature in apps which has it (like my email client...) so I'm not motivated to spend much time on it.

It's also quite complex to build since the definition of "which article is next/previous" is not quite as obvious as it sounds.

**Comment by h2oforlife** [comment]
From time to time I test Feeder for this exact feature, for me its a must have, and when I don't find it I move on. I wont click back, scroll to the next article then click it to open, when I can just swipe left and instantly see the next article. It's easier, it's faster and there's no reason to be using an app like it's 2015. The fact that it's often requested should tell you something.

**Reply by spacecowboy** [reply]
You assume I build this app for you.

I am not.

I am building the features that I think makes Feeder the best RSS app. And I let you all use it.

If you want something that I don't consider important, you are welcome to submit a PR adding that functionality.

**Reply by h2oforlife** [reply]
I don't assume that, and that's why I said I just move on, while I'm grateful for it, I don't have expectations. If it has the feature, great, if not, also great. But I wanted to point out, as you said, the feature it's often requested, and that's for a reason.
